mlir/include/mlir/IR/Diagnostics.h

```diff
@@ -644,6 +647,8 @@ class SourceMgrDiagnosticVerifierHandler : public SourceMgrDiagnosticHandler {
   void process(FileLineColLoc loc, StringRef msg, DiagnosticSeverity kind);
 
   std::unique_ptr<detail::SourceMgrDiagnosticVerifierHandlerImpl> impl;
+
+  Level level = Level::All;
```

Comment:
Please move this to the impl struct

Reply:
you can't? it's an impl struct and so the defn isn't visible outside of Diagnostics.cpp?

Reply:
In those cases expose a getter method for the Level. We shouldn't intermix pImpl and non-pimpl.